docs/content/docs/data-structures/overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Distributed Data Structures
+menuPosition: 1
+---
+
+The Fluid Framework provides developers with distributed data structures (DDSes) that automatically ensure that each
+connected client has access to the same state. The APIs provided by DDSes are designed to be familiar to programmers
+who’ve used common data structures before.
+
+When using a DDS, you can largely treat it as a local object. You can add data to it, remove data, update it, etc.
+However, a DDS is not just a local object. A DDS can also be changed by other users that are editing.
+
+Choosing the correct data structure for your scenario can improve the performance and code structure of your application.
+Below we've enumerated the data structures and described when they may be most useful.
+
+## Key-value Data
+
+These DDSes are used for storing key-value data. They are optimistic and use a last-writer-wins merge policy.
+
+* [SharedMap]({{< relref "/docs/apis/map/sharedmap.md" >}}) - a basic key-value data structure
+* [SharedDirectory]({{< relref "/docs/apis/map/shareddirectory.md" >}}) – a SharedMap with hierarchical paths instead of
+simple keys
+* [SharedCell]({{< relref "/docs/apis/cell/sharedcell.md" >}}) – a “single-object SharedMap”; useful for wrapping objects.
+
+### Key Value Scenarios
+
+Key-value data structures are the most common choice for many scenarios.
+
+* user preference data
+* current state of a survey
+* the configuration of a view
+
+### Common Issues
+
+* Storing a counter in a map will have unexpected behavior. Use the SharedCounter instead.
+* Storing a lot of data in one key-value entry may cause performance issues. Each update will update the entire value.
```

Comment:
Perf is important, but I feel like that's not the primary reason for guidance - it's the atomicity / scoping, i.e. what is changed as one atomic unit. There is no merging of sub-parts of object stored in the key.